core/src/test/java/io/cucumber/core/runner/StepDurationTimeService.java

```diff
@@ -35,4 +35,9 @@ private void handleTestStepStarted(TestStepStarted event) {
         long time = time();
         currentTime.set(time + stepDuration);
     }
+
+    @Override
+    public long elapsedTimeMillis() {
+        return 0L;
```

Comment:
This should have a similar implementation to `time`.

Reply:
> 
> 
> Looks good!
> 
> Just to be sure, you've made a PR against `develop-v5`. This version is still a bit away. You could also make the PR against `master` and we could pick it up in the next release of v4.

Ok. I will create a new PR for master. I wasn't sure if that's how it should have been so i raised one against the develop branch. But yes, its probably better to have this in v4 itself.